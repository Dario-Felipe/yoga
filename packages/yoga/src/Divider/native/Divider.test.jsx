import React from 'react';
import { render } from '@testing-library/react-native';

import { ThemeProvider, Divider } from '../..';

describe('<Divider />', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <ThemeProvider>
        <Divider />
      </ThemeProvider>,
    );

    expect(container).toMatchSnapshot();
  });
});
