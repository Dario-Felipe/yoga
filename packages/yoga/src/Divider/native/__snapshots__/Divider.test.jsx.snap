// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divider /> should match snapshot 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "flexDirection": "row",
        },
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#D7D7E0",
            "height": 1,
            "width": "100%",
          },
        ]
      }
    />
  </View>
</View>
`;
